src/components/common/YourCompany/YourCompany.vue

```diff
@@ -604,4 +613,8 @@ export default class YourCompany extends Mixins(
 #contact-info-section {
   border-bottom-left-radius: 0 !important;
 }
+
+.capitalize {
```

Comment:
There's already a Vuetify helper class for this.

https://vuetifyjs.com/en/styles/text-and-typography/#transform